pkg/operation/botanist/vpnshoot.go

```diff
@@ -77,11 +68,6 @@ func (b *Botanist) DefaultVPNShoot() (vpnshoot.Interface, error) {
 func (b *Botanist) DeployVPNShoot(ctx context.Context) error {
 	secrets := vpnshoot.Secrets{}
 
-	if !b.Shoot.ReversedVPNEnabled {
-		dhSecret := b.getDiffieHellmanSecret()
-		secrets.DH = &dhSecret
-	}
-
 	b.Shoot.Components.SystemComponents.VPNShoot.SetSecrets(secrets)
```

Comment:
Is this `SetSecrets` still used anywhere? If not, drop it from the interface together with the `Secrets` struct.